on_board_main.ipynb

```diff
@@ -0,0 +1,48 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Code to run pynq board\n",
+    "=======\n",
+    "\n",
+    "This code act as the main function file that connect our HDC code with the pynq board so that we can run the trainings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pynq\n",
+    "from pynq import Overlay\n",
+    "\n",
+    "overlay = Overlay(\"../\")\n",
+    "\n",
+    "vecadd_inst = overlay.pl_vecadd_0"
```

Comment:
We don't plan on keeping p1_vecadd in our repository, do we?

Reply:
yeah, that is the code copied from the example, but I haven't done more because the pynq package is installed on the board. It is better to develop the code on board.